src/client/components/ui/Link/Link.test.tsx

```diff
@@ -7,28 +7,50 @@ import { render, screen } from '@testing-library/react';
 
 import { Link } from '~/client/components/ui/Link/Link';
 
+import styles from './Link.module.scss';
+
+const boutonsAppearances = {
+	asPrimaryButton: styles.primary,
+	asQuaternaryButton: styles.quaternary,
+	asSecondaryButton: styles.secondary,
+	asTertiaryButton: styles.tertiary,
+};
+
 describe('Link', () => {
 	beforeAll(() => {
 		Object.defineProperty(window, 'location', {
 			value: { origin: 'https://localhost:3000' },
 		});
 	});
 	
+	describe('apparence des liens', () => {
+		Object.entries(boutonsAppearances).forEach(([appearance, expectedClass]) => {
```

Comment:
Habituellement on ne test pas le 'style', si tu penses que c'est necessaire on peut en rediscuter :) 

D'ailleurs tu as un moyen plus simple d'écrire ce test avec it.each (il y en a dans la codebase si tu veux voir)

Reply:
Je me suis dis que c'était un morceau de logique important dans le composant qui mérite que l'on vérifie qu'il fonctionne bien qui gère plusieurs cas (cf ligne 64 de Link composant), et le composant est utilisé à de nbreux endroits . T'en penses quoi ?

Reply:
ok ça me va :)